conf/scalardl-audit-custom-values.yaml

```diff
@@ -64,7 +61,6 @@ auditor:
     - name: reg-docker-secrets
 
   scalarAuditorConfiguration:
-    dbContactPoints: cassandra-lb.internal.scalar-labs.com
```

Comment:
Delete an unnecessary default value. This value is different from the `values.yaml`.
And basically, users must set contact points based on their database connection information.